InformationScripting/src/commands/CScript.cpp

```diff
@@ -85,9 +82,9 @@ Interaction::CommandResult* CScript::execute(Visualization::Item*, Visualization
 			if (args.size() > 1)
 			{
 				// $"classes g" | "script test"$
-				auto classesQuery = new AstQuery(AstQuery::QueryType::Classes, node, {"g"});
+				auto classesQuery = QueryRegistry::instance().buildQuery("classes", node, {"-s=g"});
```

Comment:
you can greatly reduce the code size of `CScript::execute` by introducing a lambda at the beginning of the method:
    
    auto query = [](auto ...args){return QueryRegistry::instance().buildQuery(args...);}

and then using it here and everywhere else in this method.